setup.cfg

```
@@ -29,11 +28,10 @@ classifiers =
 packages = find:
 install_requires =
     pyserial
```

we're missing the python-version dependent typing_extensions dependency here

```suggestion
    pyserial
    typing_extensions; python_version < '3.8'
```